### PR TITLE
Feature / Add cart notification component

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add purchase notification (replaces mini cart).
Add dependency on radix dialog.

## Justify why they are needed

I used a dialog / modal since I think that's what it really is. I was considering a toast but I think this is more accurate.
I didn't add the close button/icon just since the designs are very early stage and it works to "click outside".

I also didn't setup a very component to pull everything together in favor of a more flexible approach with individual components. When we notice we have a stable component that's well defined we can build one.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

[GRW-1265]: https://hedvig.atlassian.net/browse/GRW-1265?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #301** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/301" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/301?utm_source=stack-comment).

File: apps/store/src/components/CartNotification/Dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const contentShow = keyframes({
+  '0%': { opacity: 0, transform: 'scale(.96)' },
+  '100%': { opacity: 1, transform: 'scale(1)' },
+})
+
+const StyledOverlay = styled(DialogPrimitive.Overlay)(({ theme }) => ({
+  backgroundColor: 'rgba(0, 0, 0, 0.6)',
+  position: 'fixed',
+  inset: 0,
+  '@media (prefers-reduced-motion: no-preference)': {
```
Comment (guilhermespopolin) [inline apps/store/src/components/CartNotification/Dialog.tsx:19]: There are some implementations for CSS reset that disables animation globally so you don't need to thing checking if you're able to use them. The following snippet was taken from [here](https://piccalil.li/blog/a-modern-css-reset/)

```
/* Remove all animations, transitions and smooth scroll for people that prefer not to see them */
@media (prefers-reduced-motion: reduce) {
  *,
  *::before,
  *::after {
    animation-duration: 0.01ms !important;
    animation-iteration-count: 1 !important;
    transition-duration: 0.01ms !important;
    scroll-behavior: auto !important;
  }
}
```

Maybe we can think about adding it in our `<GlobalStyles />`. What do you thinK?

**Review by guilhermespopolin — APPROVED** [review]
Nice. It's awesome working with UI primitives like radix 🚀

**Review by robinandeer — COMMENTED** [review]
@guilhermespopolin yeah I really like it too! They should be part of the web platform 😌

**Pull request merged**